src/app/dashapp.jl

```diff
@@ -280,6 +281,12 @@ If a parameter can be set by an environment variable, that is listed as:
 - `compress::Bool`: Default ``true``, controls whether gzip is used to compress
         files and data served by HTTP.jl when supported by the client. Set to
         ``false`` to disable compression completely.
+
+- `update_title::String`: Default ``Updating...``. Configures the document.title
+        (the text that appears in a browser tab) text when a callback is being run.
+        Set to '' if you don't want the document.title to change or if you
+        want to control the document.title through a separate component or
+        clientside callback.
```

Comment:
mostly copied from 

https://github.com/plotly/dash/blob/edce37426470b900a0081969af30364e43c9a888/dash/dash.py#L243-L247

except for the `None` part.

Reply:
Only supporting `''` is fine by me.